XmlValidator/callbacks.py

```diff
@@ -87,6 +87,9 @@ def is_number(element):
             text = child.text if child.text is not None else ''
             if text[-3:] == '...':
                 text = text[:-3]
+            if '|' in text:
+                index = text.find('|')
+                text = text[:index] + text[index+1:]
```

Comment:
@HeatherW Is this essentially just replacing all pipes? I suspect it would only replace the first one but is this code just doing the same as:
  text = text.replace('|', '')